### PR TITLE
Update paloalto.md

### PR DESCRIPTION
**Pull request by CharlesLR-sekoia**

### CODE REVIEW

File: docs/integrations/paloalto.md [hunk 1]
```diff
@@ -2,55 +2,130 @@ uuid: 903ec1b8-f206-4ba5-8563-db21da09cafd
 name: Palo Alto
 
 ## Overview
+
 Palo Alto Networks offers an enterprise cybersecurity platform which provides network security, cloud security, endpoint protection, and various cloud-delivered security services.
 
-## Setup
-This setup guide will show you how to forward logs produced by your Palo Alto firewalls to SEKOIA.IO by means of an rsyslog transport channel.
+## I. Palo Alto logs
```
Comment (otetard) [inline docs/integrations/paloalto.md:8]: Title numbers are now automatically added in CSS, no longer needed to explicitly add them.

BTW, The full name of the company is “Palo Alto Networks”. I don’t know the full name of their product.

File: docs/integrations/paloalto.md [hunk 2]
```diff
@@ -2,55 +2,130 @@ uuid: 903ec1b8-f206-4ba5-8563-db21da09cafd
...
+
+On Palo Alto appliances, most of the important hardward and software activities that are relevant for security detection and analysis, are logged into three files.
+
+- Traffic: Local out traffic, Denied traffic, Allowed traffic
+- System
+- Security
+
+
+## II. Transport to the concentrator
```
Comment (otetard) [inline docs/integrations/paloalto.md:17]:
```suggestion
## II. Transport to the collector
```

File: docs/integrations/paloalto.md [hunk 3]
```diff
@@ -2,55 +2,130 @@ uuid: 903ec1b8-f206-4ba5-8563-db21da09cafd
...
+- Traffic towards the Rsyslog must be open on `UDP 514`
+
+### Configure Palo Alto
+
+#### Configure a Syslog server profile
+
+1) In the GUI, go to `Device > Serve Profiles > Syslog`.
+2) Click `Add` and enter a name for the profile such as `Syslog server`.
+3) If the firewall has more than one virtual system (vsys), select the `Location` (vsys or `Shared`) where this profile is available.
+4) Click `Add` and enter the information that the firewall requires to connect to it:
```
Comment (otetard) [inline docs/integrations/paloalto.md:31]:
```suggestion
1. In the GUI, go to `Device > Serve Profiles > Syslog`.
2. Click `Add` and enter a name for the profile such as `Syslog server`.
3. If the firewall has more than one virtual system (vsys), select the `Location` (vsys or `Shared`) where this profile is available.
4. Click `Add` and enter the information that the firewall requires to connect to it:
```

File: docs/integrations/paloalto.md [hunk 4]
```diff
@@ -2,55 +2,130 @@ uuid: 903ec1b8-f206-4ba5-8563-db21da09cafd
...
+5) Click `OK` to save the server profile.
+
+#### Configure syslog forwarding for Traffic logs
+
+1) Select `Objects > Log Forwarding`, click `Add`, and enter a `Name` to identify the profile.
+2) For each log type (here Traffic, Threat and WileFire Malicious) and each severity level, select the `Syslog server` profile and click `OK`.
+3) Select `Policies > Security` and select a policy rule.
+4) Select the `Actions` tab and select the `Log Forwarding` profile you created.
+5) In the `Profile Type` drop-down, select `Profiles` or `Groups`, and then select the security profiles or `Group Profiles` required to trigger log generation and forwarding.
+6) Select both of the `Log at Session Start` and `Log At Session End` check boxes, and click `OK`.
```
Comment (otetard) [inline docs/integrations/paloalto.md:49]:
```suggestion
1. Select `Objects > Log Forwarding`, click `Add`, and enter a `Name` to identify the profile.
2. For each log type (here Traffic, Threat and WileFire Malicious) and each severity level, select the `Syslog server` profile and click `OK`.
3. Select `Policies > Security` and select a policy rule.
4. Select the `Actions` tab and select the `Log Forwarding` profile you created.
5. In the `Profile Type` drop-down, select `Profiles` or `Groups`, and then select the security profiles or `Group Profiles` required to trigger log generation and forwarding.
6. Select both of the `Log at Session Start` and `Log At Session End` check boxes, and click `OK`.
```

File: docs/integrations/paloalto.md [hunk 5]
```diff
@@ -2,55 +2,130 @@ uuid: 903ec1b8-f206-4ba5-8563-db21da09cafd
...
+
+For detailed information about configuring a log forwarding profile and assigning the profile to a policy rule, see [Configure Log Forwarding](https://docs.paloaltonetworks.com/pan-os/8-1/pan-os-admin/monitoring/configure-log-forwarding.html#id1443a62b-8a0b-41db-a08d-5df934bf0ffc)
+
+#### Configure syslog forwarding for System and User-ID logs
 
-### Download the certificate
-In order to allow the connection of your rsyslog server to the SEKOIA.IO intake, please download the SEKOIA.IO intake certificate:
-On most linux servers, two packages need to be installed: rsyslog and rsyslog-gnutls.
+1) In the GUI, go to `Device > Log Settings`.
+2) Click each Severity level (High and Critical if also fine), select the `Syslog server` profile, and click `OK`.
```
Comment (otetard) [inline docs/integrations/paloalto.md:56]:
```suggestion
1. In the GUI, go to `Device > Log Settings`.
2. Click each Severity level (High and Critical if also fine), select the `Syslog server` profile, and click `OK`.
```

**Review by otetard — CHANGES_REQUESTED** [review]

**Review by celinequivy — APPROVED** [review]

**Review by celinequivy — APPROVED** [review]

**Pull request merged**